### PR TITLE
Improve the documentation for the case you need different node version

### PR DESCRIPTION
**Pull request by NeOMakinG**

### CODE REVIEW

File: src/content/1.7/development/compile-assets.md [hunk 1]
```diff
@@ -85,4 +85,11 @@ The "watch" build will optimize your assets for development. Please remember to
...
 
-You may be using node version 9 or 10, you should use node version 8.
+You may be using an old version or one we don't support yet. Also, while some folders (like new-theme, default, classic) support node 12.x, some may not because they are not updated yet and are waiting for a pull request to be merged.
+
+How we manage to work with this, as well as working on previous PrestaShop version, is that we use a tool to easily swap between node version.
+
+There are a lot of tools able to do this. At PrestaShop, we mainly use the ['n' package](https://www.npmjs.com/package/n?activeTab=versions) or [nvm](https://github.com/nvm-sh/nvm).
+This is pretty easy to use, when you are working on a directory which need an older or newer node version, use one of these tools to switch and then `npm install && npm run build`.
+
+PrestaShop currently have a [pull request](https://github.com/PrestaShop/PrestaShop/pull/18562) opened, after it, the classic theme will be compatible with node 12. This the last main directory that needs to be compatible with this version, after, you should be able to work with node 12.x without any problems !
```
Comment (Progi1984) [inline src/content/1.7/development/compile-assets.md:95]: @NeOMakinG This paragraph is so "instantly"... Do you remember to change the status of this paragraph after each merge ? Could you make this paragraph more general ?

File: src/content/1.7/development/compile-assets.md [hunk 2]
```diff
@@ -85,4 +85,11 @@ The "watch" build will optimize your assets for development. Please remember to
 
 If `npm install` fails with error: `Failed at the ... postinstall script.`:
 
-You may be using node version 9 or 10, you should use node version 8.
+You may be using an old version or one we don't support yet. Also, while some folders (like new-theme, default, classic) support node 12.x, some may not because they are not updated yet and are waiting for a pull request to be merged.
```
Comment (PierreRambaud) [inline src/content/1.7/development/compile-assets.md:88]: We shouldn't talk about the waiting pull request.
We should split the node versions for the classic and the admin themes.

Comment (NeOMakinG) [reply src/content/1.7/development/compile-assets.md:88]: Then, I'll just put this in WIP and wait for the PR to be merged (this should be fast..)

**Comment by NeOMakinG** [comment]
Changed to WIP in order to wait for the last update PR to be merged

**Comment by PierreRambaud** [comment]
Thanks @NeOMakinG

**Pull request merged**